intel_extension_for_transformers/llm/operator/csrc/qbits_ut/ut_utils.py

```diff
@@ -20,7 +20,14 @@
 import time
 import pytest
 from functools import wraps
-torch.ops.load_library("../build/libqbits.so")
+import sys
+
+if sys.platform.startswith("linux"):
+    torch.ops.load_library("../build/libqbits.so")
+elif sys.platform.startswith("win"):
+    torch.ops.load_library("../build/qbits.dll")
```

Comment:
will it be  qbits.pyd   when the static link come to dynamic link in windows scene ?

Reply:
we also use dynamic-link on windows